plugins/com.google.cloud.tools.eclipse.appengine.whitelist/plugin.xml

```diff
@@ -12,4 +13,14 @@
          requiredSourceLevel="1.5">
      </compilationParticipant>
   </extension>
+  
+  <extension 
+      id="blacklistedClass" 
+      name="App Engine Whitelist Marker"
+      point="org.eclipse.core.resources.problemmarkers">
```

Comment:
You could verify that the extension registry has > 0 extensions for the given extension point.

```
String definingBundleId = FrameworkUtil.getBundle(someClassInBundle).getSymbolicName();
IConfigurationElement[] elements = RegistryFactory.getRegistry().getConfigurationElementsFor(definingBundleId, extPointId);
assertThat(elements.length > 0);
```